src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -41,13 +47,13 @@ public void testIsFull() {
 		}
 
 		boolean result = rhymer.isFull();
-		Assert.assertFalse(result);
+		Assert.assertTrue(result);
 	}
 
 	@Test
 	public void testPeekaboo() {
-		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
-		final int EMPTY_STACK_VALUE = -1;
+		IntArrayStack rhymer = new IntArrayStack();
+		final int EMPTY_STACK_VALUE = 0;
```

Comment:
The original constant should be used (it would be best to use a common constant in a stacks interface).